src/components/clock.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component } from 'react';
+
+type State = {
+  time: Date;
+};
+
+type Name = {
+  name: string;
+};
```

Comment:
```suggestion
type Props = {
  name: string;
};
```
It is type for props of the component, not just name